corehq/apps/domain/views/base.py

```diff
@@ -78,6 +80,23 @@ def select(request, do_not_redirect=False, next_view=None):
         return render(request, domain_select_template, additional_context)
 
 
+@login_required
+def accept_all_invitations(request):
+    def _invite(invitation, user):
```

Comment:
This might be a nice as a helper method on `Invitation` - doesn't this same logic exist elsewhere?